drivers/docker/config.go

```diff
@@ -257,6 +257,8 @@ var (
 			hclspec.NewAttr("infra_image", "string", false),
 			hclspec.NewLiteral(`"gcr.io/google_containers/pause-amd64:3.0"`),
 		),
+
+		"disable_log_collection": hclspec.NewAttr("disable_log_collection", "bool", false),
```

Comment:
Would maybe document that this in code that this may change, unless we're committing to this being a driver option rather than eventually a client one.